docs/reference/snapshot-restore/delete-snapshot.asciidoc

```diff
@@ -0,0 +1,48 @@
+[[delete-snapshots]]
+== Delete a snapshot
+++++
+<titleabbrev>Delete snapshot</titleabbrev>
+++++
+
+////
+[source,console]
+-----------------------------------
+PUT /_snapshot/my_backup
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+
+PUT /_snapshot/my_backup/snapshot_1?wait_for_completion=true
+
+PUT /_snapshot/my_backup/snapshot_2?wait_for_completion=true
+
+PUT /_snapshot/my_backup/snapshot_3?wait_for_completion=true
+-----------------------------------
+// TESTSETUP
+
+////
+
+Use the <<delete-snapshot-api,Delete snapshot API>> to delete a snapshot
+from the repository:
+
+[source,console]
+-----------------------------------
+DELETE /_snapshot/my_backup/snapshot_1
+-----------------------------------
```

Comment:
I don't think the longer delimiters here help. I keep it to the minimum, which is four.
```suggestion
----
DELETE /_snapshot/my_backup/snapshot_1
----
```